### PR TITLE
Load assets in frontend

### PR DESCRIPTION
**Pull request by goldquest**

### CODE REVIEW

File: liveblog.module [hunk 1]
```diff
@@ -33,8 +33,8 @@ function liveblog_node_view(array &$build, Node $node, EntityViewDisplayInterfac
     $initial_number = $node->field_posts_number_initial->value;
     $load_more_number = $node->field_posts_load_limit->value;
     $javascriptSettings = [
-      'getURL'      => '/liveblog/' . $node->id() . '/posts?items_per_page=' . $initial_number,
-      'getNextURL'  => '/liveblog/' . $node->id() . '/posts?items_per_page=' . $load_more_number . '&created_op=<&created[value]=@%s',
+      'getURL'      => '/liveblog/' . $node->id() . '/posts?_format=json&sort_order=DESC&items_per_page=' . $initial_number,
+      'getNextURL'  => '/liveblog/' . $node->id() . '/posts?_format=json&sort_order=DESC&items_per_page=' . $load_more_number . '&created_op=<&created=%s',
```
Comment (milkovsky) [inline liveblog.module]: should I make sort_order=DESC default?

File: liveblog.module [hunk 2]
```diff
@@ -81,9 +81,6 @@ function template_preprocess_liveblog_posts(&$variables) {
   $node = $variables['node'];
 
   if ($node->field_status->value) {
-    // Embed related posts of the active liveblog.
-    $views_content = views_embed_view('liveblog_posts', 'liveblog_posts', $node->id());
```
Comment (milkovsky) [inline liveblog.module]: Should we remove the view in general? It is not used

**Comment by milkovsky** [comment]
AWESOME! you can merge it. I think it is time to commit the 1st version of `dist` folder as well.

**Comment by goldquest** [comment]
I will go over the code again and commit the first dist

**Pull request merged**